### DISCUSSION TITLE
Spring Liquibase compativility

### DISCUSSION THREAD

**Discussion by Freebios** [aws/aws-advanced-jdbc-wrapper#1153] [Q&A]
Hello there,

I've been struggling with setting up my spring project to work with the wrapper.
I got stuck for a while on liquibase failing to connect to my RDS Proxy, on the client side the error reported was something about the token being invalid, and on the proxy side it was simply reporting that the token wasn't including the DNS of the proxy.

After digging for a while I finally understood that liquibase was simply not using the wrapper at all. I've then found this page on how to connect using an IAM token https://docs.liquibase.com/workflows/liquibase-community/using-liquibase-and-aws-rds.html this would be a good backup solution but I'm now wondering if there is any way of directly integrating with the wrapper like I would with hikari for instance.

Thanks !

**Comment by sergiyvamz** [comment]
Hi @Freebios 

It's sad to hear you're having issues with the wrapper driver. Could you provide a bit more details about your liquibase configuration that doesn't work? 

Thank you!
